config/karma-test-shim.js

```diff
@@ -1,22 +1,42 @@
 Error.stackTraceLimit = Infinity;
```

Comment:
@nimishamukherjee @sanbornsen please follow @ppitonak advise and update this from Infiinity to something smaller.